exercises/practice/resistor-color/.meta/example.rs

```diff
@@ -0,0 +1,32 @@
+use int_enum::IntEnum;
+
+#[repr(u8)]
+#[derive(Clone, Copy, Debug, Eq, PartialEq, IntEnum)]
+pub enum ResistorColor {
+    Black = 0,
+    Brown = 1,
+    Red = 2,
+    Orange = 3,
+    Yellow = 4,
+    Green = 5,
+    Blue = 6,
+    Violet = 7,
+    Grey = 8,
+    White = 9,
+}
+
+pub fn color_to_value(color: ResistorColor) -> u8 {
+    color.int_value()
+}
+
+pub fn value_to_color_string(value: u8) -> String {
+    ResistorColor::from_int(value)
+        .map(|color| format!("{:?}", color))
+        .unwrap_or("value out of range".into())
+}
+
+pub fn colors() -> Vec<ResistorColor> {
+    (0..=9)
+        .map(|i| ResistorColor::from_int(i).unwrap_or(ResistorColor::Black))
```

Comment:
```suggestion
        .map(|i| ResistorColor::from_int(i).unwrap())
```

We know from within this file that the range `0..=9` are all resistor colors, and there are no colors outside that range, so there's no need to consider the case where that isn't true. Might be worth adding a comment to that effect as well, justifying `unwrap`.